client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -2128,4 +2133,108 @@ public void onFailure(Exception e) {
 
         assertTrue(latch.await(30L, TimeUnit.SECONDS));
     }
+
+    public void testPutStoredScript() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+
+        {
+            createIndex("index1", Settings.EMPTY);
+        }
+
+        {
+            // tag::put-stored-script-request
+            PutStoredScriptRequest request = new PutStoredScriptRequest();
+            request.id("id"); // <1>
+            request.content(new BytesArray(
+                "{\n" +
+                    "\"script\": {\n" +
+                    "\"lang\": \"painless\",\n" +
+                    "\"source\": \"Math.log(_score * 2) + params.multiplier\"" +
+                    "}\n" +
+                    "}\n"
+            ), XContentType.JSON); // <2>
+            // end::put-stored-script-request
+        }
+
+        {
+            PutStoredScriptRequest request = new PutStoredScriptRequest();
+            request.id("id");
+
+            // tag::put-stored-script-content-painless
+            XContentBuilder builder = XContentFactory.jsonBuilder();
+            builder.startObject();
+            {
+                builder.startObject("script");
+                {
+                    builder.field("lang", "painless");
+                    builder.field("source", "Math.log(_score * 2) + params.multiplier");
+                }
+                builder.endObject();
+            }
+            builder.endObject();
+            request.content(BytesReference.bytes(builder), XContentType.JSON); // <1>
+            // end::put-stored-script-content-painless
+
+
+            // tag::put-stored-script-execute
+            PutStoredScriptResponse putStoredScriptResponse = client.indices().putStoredScript(request, RequestOptions.DEFAULT);
+            // end::put-stored-script-execute
+
+            // tag::put-stored-script-response
+            boolean acknowledged = putStoredScriptResponse.isAcknowledged(); // <1>
+            // end::put-stored-script-response
+
+            assertTrue(acknowledged);
+
+            // tag::put-stored-script-execute-listener
+            ActionListener<PutStoredScriptResponse> listener =
+                new ActionListener<PutStoredScriptResponse>() {
+                    @Override
+                    public void onResponse(PutStoredScriptResponse response) {
+                        // <1>
```

Comment:
i'd rather prefer to store `response` in some kind of _ref_ and to check it after that

Reply:
nevermind - the same style is used in other documentation examples